add_remove.test.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @jest-environment jsdom
+ */
+import { addTask, deleteTask } from './src/add_remove.js';
+
+describe('adding a task to the list', () => {
+  document.body.innerHTML = '<input type="text" placeholder="Add to your list..." id="inputList">';
+  const inputList = document.getElementById('inputList');
+  inputList.value = 'Mohamed Badr';
+  const myTasks = [];
+
+  test('Add two new tasks to the list', () => {
+    addTask(myTasks);
+    addTask(myTasks);
+    expect(myTasks).toHaveLength(2);
+  });
+
+  test('Check description of first task', () => {
+    expect(myTasks[0].description).toBe('Mohamed Badr');
+  });
+
+  test('Check status of first task', () => {
+    expect(myTasks[0].completed).toBe(false);
+  });
+
+  test('Check index of second task', () => {
+    expect(myTasks[1].index).toBe(2);
+  });
+});
+
+describe('deleting a task from the list', () => {
+  const myTasks = [
+    {
+      description: 'Study Jest',
+      completed: false,
+      index: 1,
+    },
+
+    {
+      description: 'Submit Jest Project',
+      completed: false,
+      index: 2,
+    },
+
+    {
+      description: 'Get Some Sleep',
+      completed: false,
+      index: 3,
+    },
+
+    {
+      description: 'Meet Standup Team',
+      completed: false,
+      index: 4,
+    },
+
+  ];
+
+  test('Delete task with index 2', () => {
+    deleteTask(myTasks, 2);
+    expect((myTasks)).toHaveLength(3);
+  });
+
+  test('Update index after task deletion', () => {
+    expect(myTasks[2].index).toBe(3);
+  });
```

Comment:
- [ ] Very nice work so far. But I think you need to add some more tests to make it complete. 
    -  Mock a storage object to "imitate" localStorage operations.
    - Mock HTML to test if add/delete functions add or remove exactly one `<li>` element to/from the list in the DOM. 
    Kindly make changes so that localStorage methods and DOM manipulation methods related to add/delete items are tested well using Mock functions. :pray:

Reply:
Hello, I think what you are asking is for the next project, not this one.

Reply:
We still have another project that includes what you are asking for, for now, can you please make this optional, and in the next project, you will see what you asked for, thanks.

Reply:
Hello, can you please approve me and in the next project you will see what you asked for, thanks.